generated/nidaqmx/nidaqmx.proto

```diff
@@ -382,6 +382,9 @@ service NiDAQmx {
   rpc StopTask(StopTaskRequest) returns (StopTaskResponse);
   rpc TaskControl(TaskControlRequest) returns (TaskControlResponse);
   rpc TristateOutputTerm(TristateOutputTermRequest) returns (TristateOutputTermResponse);
+  rpc UnregisterDoneEvent(UnregisterDoneEventRequest) returns (UnregisterDoneEventResponse);
```

Comment:
I think we don't need to update the examples because `Unregister` is only required/encouraged if you want to reuse the task. Is that right?

Reply:
Hmm. `Unregister` is entirely optional, but it is not required if you want to reuse the task with the current event configuration/streams. You can reuse the event streams for multiple acquisitions. However, the examples currently cancel the event streams as a form of flow control. Maybe we should rethink that aspect of the examples.

Reply:
Anyway, I'm marking this resolved because no example changes are needed for this PR.